gradle.properties

```diff
@@ -6,8 +6,8 @@ POM_URL=https://github.com/airbnb/lottie-android
 POM_SCM_URL=https://github.com/airbnb/lottie-android
 POM_SCM_CONNECTION=scm:git@github.com:airbnb/lottie-android.git
 POM_SCM_DEV_CONNECTION=scm:git@github.com:airbnb/lottie-android.git
-POM_LICENSE_NAME=Apache License 2.0
-POM_LICENSE_URL=https://github.com/airbnb/lottie-android/blob/master/LICENSE
+POM_LICENSE_NAME=Apache-2.0
+POM_LICENSE_URL=https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I think we should delete the LICENSE in the repo now, right?

Reply:
Per https://www.apache.org/licenses/LICENSE-2.0 you are recommended to distribute a copy.

> ## How to apply the Apache License to your work
> 
> Include a copy of the Apache License, typically in a file called LICENSE, in your work